trifecta.cabal

```diff
@@ -82,7 +80,7 @@ library
     ghc-prim,
     hashable                    >= 1.2.1   && < 1.4,
     indexed-traversable         >= 0.1.1   && < 0.2,
-    lens                        >= 4.0     && < 6,
+    lens                        >= 4.13    && < 6,
```

Comment:
Why `4.13`. Is there a reason to not crank it up higher, to `4.19.2` or even `5`?

Reply:
No reason other than I wasn't sure what the most recent version bounds I could reasonably support are. At the very least, I needed `4.13` to remove the CPP [here](https://github.com/ekmett/trifecta/blob/37cd03a4ab249449645b344ee8fc83959153679f/src/Text/Trifecta/Highlight.hs#L29), so I opted to be conservative and make that the minimum. Perhaps someone more clever than I could do an analysis of all possible version bounds to figure out what the most recent version we could support is.

Reply:
The removal of CPP makes the bound justified: it's not an arbitrary. Let me do a quick test before this is merged though.